README.md

```diff
@@ -14,6 +14,7 @@ Generate minimaps of your code
 ## Installation
 You can install directly via pip
 ```shell
+$ pip install -U setuptools # upgrade setuptools to install minimap
```

Comment:
Why is this line important?

Reply:
because we introduce packaging with setup.cfg configuration file, and this functionality was introduced in the latest version of setuptools, so on some basic system user can have an outdated version of setuptools, this line prevent it.

Reply:
Alright, sounds good.